requirements.txt

```diff
@@ -1,22 +1,20 @@
 amqp==1.4.6
 anyjson==0.3.3
-BareNecessities==0.2.8
 billiard==3.3.0.20
 celery==3.1.18
 certifi==2015.4.28
 cookies==2.2.1
-Django==1.9
+Django==1.8.3
```

Comment:
As well here. You are behind in the latest code base you need to do an update locally